x-pack/platform/packages/shared/kbn-ai-assistant/src/chat/welcome_message_knowledge_base.test.tsx

```diff
@@ -20,13 +21,15 @@ describe('WelcomeMessageKnowledgeBase', () => {
     partial: Partial<UseKnowledgeBaseResult> = {}
   ): UseKnowledgeBaseResult {
     return {
-      isInstalling: partial.isInstalling ?? false,
-      install: partial.install ?? jest.fn(),
-      installError: partial.installError,
+      isLoading: partial.isLoading ?? false,
```

Comment:
why is this `isLoading` now?  It seems less descriptive than `isInstalling`, given we are polling and waiting for the install process to finish, and not just waiting for a request to finish "loading" something.